Sloth.Core/Services/AggieEnterpriseService.cs

```diff
@@ -26,13 +26,15 @@ public class AggieEnterpriseService : IAggieEnterpriseService
         private readonly IAggieEnterpriseClient _aggieClient;
         private readonly string _journalSource;
         private readonly string _journalCategory;
+        private readonly AggieEnterpriseOptions _options;
 
         public AggieEnterpriseService(IOptions<AggieEnterpriseOptions> options)
         {
             _aggieClient = AggieEnterpriseApi.GraphQlClient.Get(options.Value.GraphQlUrl, options.Value.Token);
 
             _journalSource = options.Value.JournalSource;
```

Comment:
Since you are overhauling options anyway, please remove these individual fields and update the dependent code to use the new _options field.